Src/Couchbase.Linq.IntegrationTests/BucketContextTests.cs

```diff
@@ -19,10 +19,8 @@ public void Test_Basic_Query()
                 where x.Type == "beer"
                 select x;
 
-            foreach (var beer in query.Take(1))
-            {
-                Console.WriteLine(beer.Name);
-            }
+            var beer = query.Take(1);
+            Assert.IsNotNull(beer);
```

Comment:
Jeff,

Looks like beer here is an IQueryable<Beer>, not an individual Beer, so IsNotNull will always be true.  I think it should use .FirstOrDefault() instead.